plugins/modules/mysql_user.py

```diff
@@ -138,6 +138,8 @@
   plugin_auth_string:
     description:
       - User's plugin auth_string (``CREATE USER user IDENTIFIED WITH plugin BY plugin_auth_string``).
+      - MariaDB If I(plugin) is ``pam`` an optional I(plugin_auth_string) ults in (``CREATE USER user IDENTIFIED WITH plugin USING plugin_auth_string``).
+        This was chosen because MySQL and MariaDB store the ``USING plugin_auth_string`` part in the ``authentication_string`` column in ``mysql.user``.
```

Comment:
```suggestion
      - MariaDB If I(plugin) is ``pam`` an optional I(plugin_auth_string) ults in (``CREATE USER user IDENTIFIED WITH plugin USING plugin_auth_string``).
        This was chosen because MySQL and MariaDB store the ``USING plugin_auth_string`` part in the ``authentication_string`` column in ``mysql.user``.
```
I would re-formulate these sentences somehow and fix a couple of typos.

Reply:
I removed the technical parts of this, now its only describing the special use case of "plugin_auth_string" for pam/auth_pam.
You're right, for a user it does not matter what SQL this generates a lang as they know how to use the module.